### PR TITLE
[fix] python module as python module

### PR DESCRIPTION
**Pull request by mosquito**

### CODE REVIEW

File: eblob-bf.spec [hunk 1]
```diff
@@ -64,6 +64,18 @@ appending records one after another.
 This package contains libraries, header files and developer documentation
 needed for developing software which uses the eblob library.
 
+%package python
```
Comment (bioothod) [inline eblob-bf.spec]: Why have you separated python?

Eblob is a very low-level library which is unlikely to be used alone, thus it is very convenient to have all needed tools in a single package, thus there is no eblob-devel, eblob-python and other packages.
Also, you have only made Fedora/RHEL package, but we also support Debian based distros.

Comment (mosquito) [inline eblob-bf.spec]: I have to use this library alone. Why not?

Comment (bioothod) [inline eblob-bf.spec]: You are likely the first person ever who does this. Anyway, what is the purpose of having python in a separate package? Not C++, not devel, but python ony?

Comment (mosquito) [inline eblob-bf.spec]: Ok, I will.

Comment (bioothod) [inline eblob-bf.spec]: The main question is - why is it needed? What is the purpose?

Comment (mosquito) [inline eblob-bf.spec]: Directory named as "bindings". I think this is right. Each bindings has a package.

Comment (bioothod) [inline eblob-bf.spec]: You want to force all users to change their dependencies, to install multiple packages, to monitor multiple packages and versions during updates, to try not to break update with invalid version, and all users will still install all packages, not one, but 3 anyway.

All of this just for 'I think this is right'?

This is convenient to have as little as possible infrastructure packages - infrastructure is designed to be self-hosting. Eblob is such a package. When one needs a low-level storage (like elliptics does) admin should not think about what has to be installed among 3 packages, there is just a single button to press.

Please do not break this convenience without major need.

**Pull request closed without merging**